src/core/include/openvino/op/reorg_yolo.hpp

```diff
@@ -35,6 +35,9 @@ class OPENVINO_API ReorgYolo : public Op {
     Strides get_strides() const {
         return m_strides;
     }
+    void set_strides(const Strides& strides) {
```

Comment:
This setter is also defined in #18633 but input is `size_t` type  as all strides are same. I think will be simper to set from GPU without vector creation.
Move implementation to *.cpp file.
If setter done as vector consider to change it as 
```cpp
 void set_strides(Strides strides){
    m_strides = std::move(strides);
}

Reply:
First of all, I changed this setter as your suggestion. But once this PR is merged first, it seems that it needs to be updated again in your PR through rebase.